golang/personStruct/personStruct.go

```diff
@@ -0,0 +1,59 @@
+package personStruct
+
+import "fmt"
+
+type person struct {
+	name string
+	age int
+	gender string
+}
+
+func (p person) GetName() string {
+	return p.name
+}
+
+func (p person) GetAge() int {
+	return p.age
+}
+
+func (p person) GetGender() string {
+	return p.gender
+}
+
+func (p *person) SetAge(age int) error {
+	if age > 0 && age < 122 {
+		p.age = age
+		return nil
+	}
+	return fmt.Errorf("age is higher or lower than bound")
```

Comment:
const ErrWrongAge = errors.New("age is higher or lower than bound")
```suggestion
	return ErrWrongAge
```